packages/config-array/package.json

```diff
@@ -23,7 +23,9 @@
   },
   "homepage": "https://github.com/eslint/rewrite#readme",
   "scripts": {
-    "build": "rollup -c",
+    "build:dedupe-types": "node ../../tools/dedupe-types.js dist/cjs/index.cjs dist/esm/index.js", 
+    "build": "rollup -c && npm run build:dedupe-types && tsc -p tsconfig.esm.json && tsc -p tsconfig.cjs.json",
```

Comment:
We need to either delete the copied `types.ts` at the end of the build, or rename it to `.d.ts` when we copy it to avoid generating declarations of statements.

Reply:
Sorry, I'm not sure I follow. This outputs both `types.ts` and `types.d.ts` in the output directories, unless I'm missing something?

Reply:
Yes, `types.ts` is not required at publish time.

Reply:
Gotcha. I think we can solve that by using `files` in `package.json` to avoid adding another step to this process.

Reply:
> Gotcha. I think we can solve that by using `files` in `package.json` to avoid adding another step to this process.

Should we then do the same in the `object-schema` package?

Reply:
Yes, I'll update all the packages in a separate PR.